litgpt/scripts/convert_hf_checkpoint.py

```diff
@@ -4,22 +4,35 @@
 import json
 from collections import defaultdict
 from functools import partial
+import os
 from pathlib import Path
 from pprint import pprint
 from typing import Dict, List, Optional, Tuple, Union
 
+from tqdm import tqdm
 import torch
 from lightning.fabric.utilities.load import _NotYetLoadedTensor as NotYetLoadedTensor
 
 from litgpt import Config
 from litgpt.utils import extend_checkpoint_dir, incremental_save, lazy_load, save_config
 
 
+def estimate_total_entries(bin_files):
```

Comment:
Is it used anywhere?

Reply:
Thanks, that's a leftover, I will clean this up.